lib/validate.js

```diff
@@ -120,6 +120,24 @@ const validateCategory = (
   const schemaProps = new Set(Object.keys(schema));
   const objectProps = new Set(Object.keys(instance));
   const props = new Set([...schemaProps, ...objectProps]);
+
+  const validate = (value, prop, definition) => {
+    const opts = { ...options };
+
+    let err;
+    if (definition.domain) {
+      opts.path = `${path}${prop}`;
+      err = ms.validate('domains', definition.domain, value, opts);
+    } else if (definition.category) {
+      opts.path = `${path}${prop}.`;
+      err = ms.validate('category', definition.category, value, opts);
+    }
```

Comment:
`opts.path` is repeated in both cases

```javascript
const opts = { ...options };
opts.path = `${path}${prop}`;

let err;
if (definition.domain) {
  err = ms.validate('domains', definition.domain, value, opts);
} else if (definition.category) {
  err = ms.validate('category', definition.category, value, opts);
}
```

Reply:
In the case of a domain, we have a dot at the end.